scripts/postdeploy.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+cd app
+
+# Run Django migrations.
+python manage.py migrate
+# Seed database with dummy users
+python manage.py seedusers
```

Comment:
If this kicks in for the production instance, any github miner now has working creds for a prod database. For security reasons, you don't want to "bootstrap some users" outside of testing, which happens in a separate database that gets destroyed once testing finishes.